src/main/java/de/tum/cit/aet/artemis/atlas/api/AbstractAtlasApi.java

```diff
@@ -0,0 +1,6 @@
+package de.tum.cit.aet.artemis.atlas.api;
+
+import de.tum.cit.aet.artemis.core.api.AbstractApi;
+
+public abstract class AbstractAtlasApi implements AbstractApi {
```

Comment:
What are you planning to add in this class? Is this class necessary or helpful?

Reply:
the idea was to give some structure, even though they don't provide anything useful, yet. In the future, we consider adding information on whether the module is activated.